DASCharter-2020.html

```diff
@@ -1,677 +1,1346 @@
+          The WG will reuse existing security and privacy-focused web features
+          and APIs developed in other groups where applicable. If the existing
+          solutions do not provide the level of protection required, the WG
+          will explore innovative new security and privacy mechanisms.
+        </p>
+        <div id="section-out-of-scope">
+          <h3 id="out-of-scope">
+            Out of Scope
+          </h3>
+          <p>
+            Hardware security services are out of scope for this WG.
+          </p>
+          <p>
+            Priority will be given to developing simple and consensual APIs,
+            leaving more complex features to future versions.
+          </p>
+        </div>
+      </div>
+      <div class="deliverables">
+        <h2 id="deliverables">
+          Deliverables
+        </h2>
+        <p>
+          The Devices and Sensors WG will follow a <a href=
+          "https://www.w3.org/2019/02/testing-policy.html">test as you
+          commit</a> approach to specification development, for specifications
+          in CR or above.
+        </p>
+        <blockquote cite="https://www.w3.org/2019/02/testing-policy.html">
+          <p>
+            All normative spec changes are generally expected to have a
+            corresponding pull request in <a href=
+            "https://github.com/w3c/web-platform-tests">web-platform-tests</a>,
+            either in the form of new tests or modifications to existing tests,
+            or must include the rationale for why test updates are not required
+            for the proposed update.
+          </p>
+          <p>
+            Typically, both pull requests (spec updates and tests) will be
+            merged at the same time. If a pull request for the specification is
+            approved but the other needs more work, add the '<a href=
+            "https://w3c.github.io/spec-labels.html" rel="nofollow">needs
+            tests</a>' label or, in web-platform-tests, the '<a href=
+            "https://github.com/w3c/web-platform-tests/issues?utf8=%E2%9C%93&amp;q=label%3Astatus%3Aneeds-spec-decision%20">status:needs-spec-decision</a>'
+            label. Note that a test change that contradicts the specification
+            should not be merged before the corresponding specification change.
+          </p>
+          <p>
+            If testing is not practical due to web-platforms-tests limitations,
+            please explain why and if appropriate file an issue with the
+            '<a href=
+            "https://github.com/w3c/web-platform-tests/issues?utf8=%E2%9C%93&amp;q=label%3Atype%3Auntestable%20">type:untestable</a>'
+            label to follow up later.
+          </p>
+        </blockquote>
+        <h3 id="rec-track">
+          Normative Specifications
+        </h3>
+        <p>
+          The Working Group will deliver the following W3C normative
+          specifications:
+        </p>
+        <h4>
+          Sensors
```

Comment:
I suggest that we remove this `h4` because some specs (like Battery Status) are not sensor specs.

Reply:
3d67c77